script.js

```diff
@@ -0,0 +1,97 @@
+var script = document.createElement('script');
+script.src = "https://checkout.razorpay.com/v1/checkout.js";
+
+window.onload  = function(e)
+{
+    document.body.appendChild(script);
+
+    script.onload = function()
+    {
+        var script2  = document.createElement("script");
+
+        var div = document.createElement('div');
+
+        var html = "<form id ='razorpayform' name='razorpayform' method='POST'> " +
```

Comment:
Why not return HTML as usual from PHP and let the script handle just the dynamic parts? Moving HTML to JS isn't clean.